Sources/Repeat/Repeater.swift

```diff
@@ -187,7 +187,8 @@ open class Repeater : Equatable {
 	private var queue: DispatchQueue? = nil
 	
 	/// Unique identifier
-	public let id: UUID = UUID()
+  @available(*, deprecated, message: "Please use the equal-to operator (==) instead")
+	public let id = UUID()
```

Comment:
@malcommac If it were me, I would probably just remove the `id` property altogether as it isn't being used elsewhere inside `Repeater`. But since it is part of the public API, it might be best to deprecate it first and then remove it in a future release. I marked it as deprecated, but let me know if you'd like me to remove it altogether. If a consumer really needs to associate a `Repeater` object with a `UUID`, they could easily do it themselves using a dictionary.

Reply:
You are absolutely right, I'm ok with deprecation and I'll remove it as before the 1.0.
Thank your pointing me to this solution, I completely forgot the `===` operator.